cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/TaskWorkerService.java

```diff
@@ -64,7 +68,7 @@
       .setExecThreadPoolSize(cConf.getInt(Constants.TaskWorker.EXEC_THREADS))
       .setBossThreadPoolSize(cConf.getInt(Constants.TaskWorker.BOSS_THREADS))
       .setWorkerThreadPoolSize(cConf.getInt(Constants.TaskWorker.WORKER_THREADS))
-      .setHttpHandlers(new TaskWorkerHttpHandlerInternal(this.cConf, this::stopService));
+      .setHttpHandlers(new TaskWorkerHttpHandlerInternal(this.cConf, hConf, this::stopService));
```

Comment:
You don't new with a constructor that marked with `@Inject`. You either new or use guice injection, not a mix.

Reply:
Removed `@Inject`